src/cdk/tree/nested-node.ts

```diff
@@ -40,7 +40,7 @@ import {getTreeControlFunctionsMissingError} from './tree-errors';
  *     {{node.name}}
  *      <cdk-nested-tree-node>{{child1.name}}</cdk-tree-node>
  *      <cdk-nested-tree-node>{{child2.name}}</cdk-tree-node>
```

Comment:
The open and close tags here are also different, maybe you could correct these too?

Reply:
Good catch! I've got it.